views/index.py

```diff
@@ -325,6 +325,18 @@ def reserve(copy_id):
     return redirect(url_for('library.index'))
 
 
+@library.route('/check_reservation_status_db')
+def check_reservation_status_db():
+    db.session.query(RentalLog)\
+        .filter(RentalLog.book_status == BookStatus.RESERVED)\
+        .filter(RentalLog._reservation_end > datetime.now(tz=pytz.utc))\
+        .update({RentalLog.book_status: BookStatus.RETURNED})
+    db.session.query(Copy)\
+        .filter(Copy.available_status == False)\
+        .update({Copy.available_status: True})
```

Comment:
You can't just globally change status of all copies to True.
We should just change the copies, which rental has just changes.